README.md

```diff
@@ -1,7 +1,7 @@
+![](https://github.com/shafreenAnfar/synapse/workflows/Synapse%20Daily%20Build/badge.svg)
```

Comment:
This points to your forked github repo build. can you please change it?

Reply:
Done!

Reply:
@shafreenAnfar,
Why there are no tag nodes 
Example:- "![CI]"

Can you please take a screenshot of the preview changes?